sdk/js/src/utils/consts.ts

```diff
@@ -611,3 +611,7 @@ function evm_chain_subset(e: EVMChainName): ChainName {
   // will fail to compile if 'e' can't be typed as a [[ChainName]]
   return e;
 }
+
+export const MAINNET_ALPH_MINIMAL_CONSISTENCY_LEVEL = 105
+export const TESTNET_ALPH_MINIMAL_CONSISTENCY_LEVEL = 105
```

Comment:
Let's use a smaller number for testnet? maybe `10` as you initially set.

Reply:
Done